src/components/Checkout.js

```diff
@@ -45,8 +45,15 @@ class Checkout extends Component {
     // Do this or the trace won't include the backend transaction
     Sentry.configureScope(scope => scope.setSpan(transaction));
 
+    let se, customerType, email
+    Sentry.withScope(function(scope) {
+      [ se, customerType ] = [scope._tags.se, scope._tags.customerType ]
+      email = scope._user.email
+    });
+
     let response = await fetch(`${BACKEND}/checkout`, {
       method: "POST",
+      headers: { se, customerType, email },
```

Comment:
I recorded a TODO for using a requests library where we can hook into it to set these headers, so we put all the code there for parsing the tags, email, and setting it as headers too.